build.fsx

```diff
@@ -266,14 +272,14 @@ Target "CodeGen.NetCore" (fun _ ->
 )
 
 Target "Build.NetCore" (fun _ ->
-    [ "src/fsharp/FSharp.Compiler.Service/";
-      "src/fsharp/FSharp.Compiler.Service.ProjectCracker/";
-      "src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool/" ]
-    |> List.iter (run "dotnet" "-v pack %s -c Release")
+    [ "src/fsharp/FSharp.Compiler.Service.netcore/";
+      "src/fsharp/FSharp.Compiler.Service.ProjectCracker.netcore/";
+      "src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool.netcore/" ]
+    |> List.iter (run "dotnet" "pack %s -v detailed -c Release")
```

Comment:
let's use just `-v n` because `d` is too verbose. 
`n` give flow and fsc args, and is enough